pkg/gatherers/clusterconfig/operators_pods_and_events.go

```diff
@@ -313,6 +314,34 @@ func getContainerLogs(ctx context.Context,
 	return records
 }
 
+// getLogWithStacktracing search for the first stack trace line and offset it by logLinesOffset
+func getLogWithStacktracing(logArray []string) string {
+	var log []string
+	var found bool
+
+	for idx := range logArray {
+		line := logArray[idx]
+		// if it already found the stack we just want to keep add the log till the end
+		if found {
+			log = append(log, line)
+			continue
+		}
+		// tries to find the stack message at the given line and add it to the log
+		if found = stackTraceRegex.MatchString(line); found {
+			log = append(log, line)
+			continue
+		}
+		// if it didn't find the line yet, keep add it as offset limiting by the arbitrary
+		// offset limit value
+		log = append(log, line)
+		if len(log) > logLinesOffset {
+			log = log[1:]
+		}
+	}
+
+	return strings.Join(log, "\n")
```

Comment:
OK so I was looking at this again. It works fine, but maybe something like following would be more straightforward:
```

	for idx := range logArray {
		line := logArray[idx]
		log = append(log, line)
		if !found {
			if len(log) > logLinesOffset {
				log = log[1:]
			}
			found = stackTraceRegex.MatchString(line)
		}
	}
```
WDYT?

Reply:
Looks good.